c8run/internal/health/camunda.go

```diff
@@ -66,13 +67,14 @@ func isRunning(name, url string, retries int, delay time.Duration) bool {
 }
 
 func PrintStatus(settings types.C8RunSettings) error {
-	operatePort, tasklistPort, identityPort := 8080, 8080, 8080
+	operatePort, tasklistPort, identityPort, camundaPort := 8080, 8080, 8080, 8080
 
 	// Overwrite ports if Docker is enabled
 	if settings.Docker {
 		operatePort = 8081
 		tasklistPort = 8082
 		identityPort = 8084
+		camundaPort = 8085
```

Comment:
8085 is the inbound connectors port.  the v2 api port could be anything depending on which component you're trying to reach. 8088 would refer to the zeebe gateway restapi, and is probably the most reasonable endpoint to list here.

Reply:
Thanks! Adjusted!